recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend

```diff
@@ -0,0 +1,28 @@
+################################################################################
+# We only need Xen tools, so we can start domains
+################################################################################
+XEN_REL = "4.9"
```

Comment:
Are you sure, you will not to forget to change this, when we'll upgrade XEN to the next version?

Reply:
this is the way meta-virtualization works for Xen, won't fix